orchestration/templates/ingest-raw-archive.yaml

```diff
@@ -0,0 +1,117 @@
+apiVersion: argoproj.io/v1alpha1
+kind: WorkflowTemplate
+metadata:
+  name: ingest-raw-archive
+spec:
+  entrypoint: main
+  serviceAccountName: {{ .Values.serviceAccount.k8sName }}
+  templates:
+    ##
+    ## Entrypoint for ingesting a raw ClinVar archive into the Jade repository.
+    ##
+    ## Ingests the raw XML, along with a row in the release_history table to track it.
+    ## Does not run the ingest if the XML already exists.
+    ##
+    - name: main
+      inputs:
+        parameters:
+          - name: gcs-prefix
+          {{- $gcsPrefix := "{{inputs.parameters.gcs-prefix}}" }}
+          - name: release-date
+          {{- $releaseDate := "{{inputs.parameters.release-date}}" }}
+      dag:
+        tasks:
+          # FIXME: This logic still needs reworked to truly be idempotent.
```

Comment:
This was the original motivation for the ticket. I'm going to cut a new branch and start working on it once this is stable in dev.